cupyx/scipy/ndimage/measurements.py

```diff
@@ -319,9 +319,10 @@ def variance(input, labels=None, index=None):
                            cupy.float64, cupy.uint32, cupy.uint64,
                            cupy.ulonglong]:
         warnings.warn(
-            'Using the slower implementation as '
-            'cupyx.scipy.ndimage.sum supports int32, float16, '
-            'float32, float64, uint32, uint64 as data types'
+            'Using the slower implementation because the provided '
+            'type is not supported by cupyx.scipy.ndimage.sum. '
```

Comment:
```suggestion
            f'type {input.dtype} is not supported by cupyx.scipy.ndimage.sum. '
```
Like that @emcastillo ?  (-: